.gitmodules

```diff
@@ -36,3 +36,6 @@
 [submodule "src/tools/clippy"]
 	path = src/tools/clippy
 	url = https://github.com/rust-lang-nursery/rust-clippy.git
+[submodule "src/tools/rustfmt"]
+	path = src/tools/rustfmt
+	url = git@github.com:rust-lang-nursery/rustfmt.git
```

Comment:
Could this use an https url like the other submodules? I don't know why the others don't use ssh, bug may as well be consistent I guess?